wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,72 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+Users are getting banned for several reasons not limited to:
+-   Continuous misbehaviour regardless of how many infringements were given.
+-   Sharing your account with other people.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What is considered cheating?
+The rules says that any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
 
-This is a one-strike rule; cheaters will be **permanently banned.**
+This is a one-strike rule; cheaters will be *permanently banned*.
 
-I have been banned for cheating. I didn't cheat!
-------------------------------------------------
 
+## I have been banned for cheating. I didn't cheat!
 osu! uses an automated metrics combined with manual confirmation.
 
-The staff never places an incorrect ban for cheating. Before they do any actions, they make sure it first 100%.
+The staff never places an incorrect ban for cheating. Before they take any actions, they make sure it first 100%.
```

Comment:
replace "staff" with "team"

"they make sure it first 100%" isn't that missing a "is" somewhere ?